server/routers/data/state-of-play.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+var ukParties = require('uk-political-parties');
+var _ = require('lodash');
+
+var knownParties = [
```

Comment:
Could this be something you add to the uk-political-parties library?

e.g. 
* `ukParties.getKnownParties()` return an array of IDs. and
* `ukParties.isKnownParty(id)` returns a boolean
*  `ukParties.isOther(id or name)` returns boolean